src/tir/schedule/primitive/reduction.cc

```diff
@@ -938,7 +938,9 @@ StmtSRef RFactor(ScheduleState self, const StmtSRef& rf_loop_sref, int factor_ax
   Block new_scope_root_block = BlockReplacer::Replace(
       old_scope_root_block, rf_body, loops[0], wb_block_creator.new_block_realize_, block_realize,
       GetRef<For>(rf_loop), reduce_loop_vars, loop_vars2loop, rf_buffer);
-  self->Replace(scope_root, new_scope_root_block, {{old_scope_root_block, new_scope_root_block}});
+  self->Replace(
+      scope_root, new_scope_root_block,
+      {{old_scope_root_block, new_scope_root_block}, {block, wb_block_creator.new_block_}});
```

Comment:
This is a bug fix. @junrushao1994 Do you think I can fix it here or send a separate PR?

Reply:
Hmm given it’s really minor (forgot to add a field) and nobody is affected by this bug right now, we could do it in this PR